docs/_posts/2021-02-15-42.2.19-release.md

```diff
@@ -0,0 +1,32 @@
+---
+title: PostgreSQL JDBC Driver 42.2.19 Released
+date: 2021-02-15 19:19:40 +0000
+categories:
+  - new_release
+version: 42.2.19
+---
+**Notable changes**
+
+
+**Notable Changes**
+- Now the driver use SASLprep normalization for SCRAM authentication fixing some issues with spaces in passwords.
```

Comment:
Typo for "use" vs "uses". How about?

The driver now uses SASLprep normalization for SCRAM authentication which fixes some issues with whitespace in passwords.